tools/cldr-apps/src/main/webapp/js/esm/cldrSurvey.js

```diff
@@ -180,6 +181,13 @@ function handleChangedLocaleStamp(stamp, name) {
   if (stamp <= surveyNextLocaleStamp) {
     return;
   }
+  /*
+   * For performance, postpone the all-row WHAT_GETROW update if multiple
+   * requests (e.g., vote or single-row WHAT_GETROW requests) are pending.
+   */
+  if (cldrVote.pending()) {
+    return;
```

Comment:
Before https://github.com/unicode-org/cldr/pull/1109 we had ` if (cldrAjax.queueCount() > 1) return` here.